### PR TITLE
Add acknowlegements to README

### PR DESCRIPTION
**Pull request by dominiquesydow**
## Description

Add acknowlegements (funders, external resources, contributors).

## TODOs

- [x] Add "Acknowlegements" section to README
- [x] Move "Funding" section here
- [x] Add "External resources" section (packages, databases) + links to website (main project website)
- [x] Add "Contributors" section: Everyone how has helped (worked on talktorials, helped with infrastructure, fixed a typo) > remove list and add sentence "contributors are listed in the notebooks" + link to GH contributors

## Questions

None

## Status

- [x] Ready to go

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,7 +93,49 @@ It will help measure the impact of the TeachOpenCADD platform and future funding
...
+- Documentation: 
+  [`sphinx`](https://www.sphinx-doc.org), 
+  [`nbshinx`](https://nbsphinx.readthedocs.io)
+- Code style: 
+  [`black-nb`](https://github.com/tomcatling/black-nb)
+
+#### Databases
+
+- [ChEMBL](https://www.ebi.ac.uk/chembl/)
+- [PDB](https://www.rcsb.org/)
```
Comment (jaimergp) [inline README.md:134]: I think the official name is RCSB PDB?

File: README.md [hunk 2]
```diff
@@ -93,7 +93,49 @@ It will help measure the impact of the TeachOpenCADD platform and future funding
...
+  [`nglview`](http://nglviewer.org/nglview/latest/)
+- Web services clients: 
+  [`pypdb`](https://github.com/williamgilpin/pypdb), 
+  [`chembl_webresource_client`](https://github.com/chembl/chembl_webresource_client)
+- Continuous integration: 
+  [`pytest`](https://docs.pytest.org), 
+  [`nbval`](https://nbval.readthedocs.io)
+- Documentation: 
+  [`sphinx`](https://www.sphinx-doc.org), 
+  [`nbshinx`](https://nbsphinx.readthedocs.io)
```
Comment (jaimergp) [inline README.md:127]:
```suggestion
  [`nbsphinx`](https://nbsphinx.readthedocs.io)
```

File: README.md [hunk 3]
```diff
@@ -93,7 +93,49 @@ It will help measure the impact of the TeachOpenCADD platform and future funding
...
+  [`openbabel`](http://openbabel.org/wiki/), 
+  [`mdanalysis`](https://www.mdanalysis.org/), 
+  [`biopython`](https://biopython.org/), 
+  [`biopandas`](http://rasbt.github.io/biopandas/), 
+  [`opencadd`](https://opencadd.readthedocs.io/en/latest/)
+- Data science (PyData stack): 
+  [`numpy`](https://numpy.org/), 
+  [`pandas`](https://pandas.pydata.org/), 
+  [`scikit-learn`](https://scikit-learn.org/), 
+  [`juypter`](https://jupyter.org/),
```
Comment (jaimergp) [inline README.md:113]:
```suggestion
  [`jupyter`](https://jupyter.org/), 
```

**Review by jaimergp — APPROVED** [review]
LGTM! Thanks!

File: README.md (hunk 3 above)
Comment (dominiquesydow) [reply README.md:113]: That's a typo, I do over and over again - thanks for spotting this!

**Comment by dominiquesydow** [comment]
@AndreaVolkamer - could you please check out the "Contributors" section? 

I replaced the name list with 
> Many thanks to everyone who has contributed to TeachOpenCADD by working on talktorials (check out the talktorial READMEs - [example](https://github.com/volkamerlab/teachopencadd/tree/packaging/teachopencadd/talktorials/001_query_chembl)) and/or by helping in any other way (see [GitHub contributors](https://github.com/volkamerlab/teachopencadd/graphs/contributors)).

**Comment by AndreaVolkamer** [comment]
Looks great, one suggestion, maybe change/add
* `Many thanks to everyone who has contributed to TeachOpenCADD by working on talktorials (check out the talktorial READMEs ...` 
* to (or similar) `TeachOpenCADD has been initiated by the members of [volkamerlab](https://volkamerlab.org/), Charité Universitätsmedizin Berlin, special thanks to @dominiquesydow, @jaimergp and @AndreaVolkamer, and filled with life by our students from the CADD courses taught in the bioinformatics studies at FU Berlin. Many thanks to everyone who has contributed to TeachOpenCADD by working on talktorials (check out the info on top of each talktorial's README for author information ...`

**Comment by AndreaVolkamer** [comment]
Maybe shorten funding to sth like `Note: Volkamerlab's projects are supported by several public funding sources (for more info see our [webpage](https://volkamerlab.org/)).`

**Review by AndreaVolkamer — APPROVED** [review]
Perfekt! Ready to go!

**Pull request merged**